mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/entity/EntityHelperSelectStartMethodGenerator.java

```diff
@@ -56,8 +56,13 @@ public EntityHelperSelectStartMethodGenerator(
                 QueryBuilder.class)
             .addCode("$[return selectFrom");
 
-    for (PropertyDefinition property : entityDefinition.getAllColumns()) {
-      selectStartBuilder.addCode("\n.column($L)", property.getCqlName());
+    for (PropertyDefinition property : entityDefinition.getAllValues()) {
+      if (property.getSelector().equals(property.getCqlName())) {
```

Comment:
Nit: this does not clearly indicate that we distinguish computed values.

We could either introduce `PropertyDefinition.isComputed()`, or iterate `getAllColumns` and `getComputedValues` separately.

Reply:
That's actually how I previously had it, my reasoning for changing it was in possible anticipation of aliasing not necessarily being bound to computed properties, although that seems rather unlikely.

it probably is better to make it more explicit that this is why things are being aliases here, I"ll change it back.